lib/middleware.ts

```diff
@@ -189,5 +225,17 @@ var formatLine = (options_arg) => {
 if (process.env.NODE_ENV === "test") {
   module.exports = (clever_options, morgan_options) => morgan(formatLine(clever_options), morgan_options);
 } else {
-  module.exports = (clever_options) => morgan(formatLine(clever_options), {stream: process.stderr});
+  module.exports = (clever_options, context_logger_options = defaultContextLoggerOpts) => {
+    // `source` is the one required field
+    if (!clever_options.source) {
+      throw new Error("Missing required config for 'source' in Kayvee middleware 'options'");
+    }
+    const logger = new kayveeLogger(clever_options.source);
+    return (req, res, next) => {
+      if (context_logger_options.enabled) {
+        req.log = new ContextLogger(logger, context_logger_options.handlers, req);
```

Comment:
Good point. What if there is already a `req.log` should we call it `kayvee`? Is there a standard method to add request bound objects in express?